packages/jest-resolve-dependencies/src/index.ts

```diff
@@ -58,8 +58,12 @@ class DependencyResolver {
           dependency,
           options,
         );
-      } catch (e) {
-        resolvedDependency = this._resolver.getMockModule(file, dependency);
+      } catch (e1) {
+        try {
+          resolvedDependency = this._resolver.getMockModule(file, dependency);
+        } catch (e2) {
+          resolvedDependency = null;
```

Comment:
This is true, but I personally feel that it's more explicit to do something in this catch block, and make it clear what the expected result in this case would be.

If you feel that it would be cleaner not to do the assignment, that should also work just fine.

Reply:
It's perfectly fine to leave the catch block empty and put a comment inside :)

Reply:
Then I'll do that!